delta/plugins/blazegraph/src/test/resources/serialization/view-tag-added.json

```diff
@@ -11,5 +11,8 @@
   },
   "tag" : "mytag",
   "targetRev" : 1,
-  "uuid" : "f8468909-a797-4b10-8b5f-000cba337bfa"
+  "uuid" : "f8468909-a797-4b10-8b5f-000cba337bfa",
+  "tpe": {
```

Comment:
this encoding looks a bit odd, and there are inconsistencies in the value structure (`ElasticSearch` vs `IndexingBlazegraphView`)

would something like `"tpe": "IndexingElasticSearchView"` work better?

Reply:
or maybe: `viewType` instead...

Reply:
Do we care about that> This is just going to the DB.

Reply:
if we're going to have to migrate these events later, it would be very good to have a reasonable consistent representation.